core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -1444,9 +1522,14 @@ public void updateViewLocationConflict() {
     catalog().dropView(identifier);
     assertThat(catalog().viewExists(identifier)).as("View should not exist").isFalse();
 
+    String expectedMessage =
+        catalog() instanceof RESTCatalog ? "View does not exist: ns.view" : "Cannot commit";
+    Class<?> expectedException =
+        catalog() instanceof RESTCatalog ? NoSuchViewException.class : CommitFailedException.class;
```

Comment:
can you please move the changes in viewCatalogTests to the separate PR and merge it first? 
I am working on View support for Nessie catalog and these tests are failing for me too. 

Also I think we cannot have a catalog specific check in this class. It will become too clumsy once we extend this test for all the catalogs. Can you please simplify the assert to fail on generic reason?

Reply:
done in https://github.com/apache/iceberg/pull/8865